docs/usage.md

```diff
@@ -194,6 +194,10 @@ genome: 'GRCh37'
 
 You can also generate such `YAML`/`JSON` files via [nf-core/launch](https://nf-co.re/launch).
 
+### If using Simpleaf
+
+When building reference index, the underlying mapper of Simpleaf operates on a large number of temporary files on disk simutaneously. Therefore, for environments where CPUs and disk I/O are limited, such as on AWS, specifying `scratch=true` for the `SIMPLEAF_INDEX` module in the `conf/modules.config` file is necessary to avoid slowdowns and potential failures.
+
```

Comment:
give it as an example, e.g.
````suggestion
```nextflow title="conf/modules.config"
withNAME: SIMPLEAF_INDEX {
  scratch=true
}
```
````
or something similar.

Reply:
addressing in: https://github.com/nf-core/scrnaseq/pull/443